src/analyserwaveform.cpp

```diff
@@ -16,6 +16,7 @@ AnalyserWaveform::AnalyserWaveform(ConfigObject<ConfigValue>* pConfig) :
         m_skipProcessing(false),
         m_waveformData(NULL),
         m_waveformSummaryData(NULL),
+        m_stride(),
```

Comment:
For now you have to call m_stride.init(0,0);
@rryan : When can we stating to use c++11 code?